benefits/eligibility/views.py

```diff
@@ -64,6 +64,9 @@ def index(request):
             verifier = EligibilityVerifier.objects.get(id=verifier_id)
             session.update(request, verifier=verifier)
 
+            types_to_verify = agency.type_names_to_verify(verifier)
+            analytics.selected_verifier(request, verifier.name, types_to_verify)
```

Comment:
Since `benefits.core.analytics.Event` updates the [event properties](https://github.com/cal-itp/benefits/pull/1057/files#diff-caee5b8a703ead64243ea99814f85560027016eef535273ace62604eaa148101R56) and [user properties](https://github.com/cal-itp/benefits/pull/1057/files#diff-caee5b8a703ead64243ea99814f85560027016eef535273ace62604eaa148101R71) with the session verifier name, is it redundant to pass in `verifier.name` here?

Reply:
Especially that we have a test to ensure the base event grabs the `verifier`... I think you're right, it can be removed.

Reply:
Done in [be44936](https://github.com/cal-itp/benefits/pull/1057/commits/be44936249cb20225239d6ce104f3aa0c64d976c)